src/ExperimentTable.js

```diff
@@ -60,6 +60,15 @@ class ExperimentTable extends React.Component {
       )
   }
 
+  _isContains(data, value) {
```

Comment:
This function has a very confusing name. Could you please think of another one? Also add a one sentence description of what it does (or a simple example of when it should either `true` or `false`).